src/main/java/org/jitsi/videobridge/RtpChannel.java

```diff
@@ -1726,13 +1726,38 @@ public void setSources(List<SourcePacketExtension> sources)
         addedSSRCs.removeAll(oldSignaledSSRCs);
         if (!addedSSRCs.isEmpty())
         {
+
+            boolean addToSyncronizer = false;
+            Recorder recorder = getContent().getRecorder();
```

Comment:
Calling getRecorder will create a Recorder instance if one doesn't exist (regardless of whether recording is enabled or not). It should only be called if isRecording() returns true (yeah, the design of the API is not good...).